src/main/java/com/cleanroommc/groovyscript/compat/mods/essentialcraft/DemonTradeManager.java

```diff
@@ -0,0 +1,87 @@
+package com.cleanroommc.groovyscript.compat.mods.essentialcraft;
+
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.Example;
+import com.cleanroommc.groovyscript.api.documentation.annotations.MethodDescription;
+import com.cleanroommc.groovyscript.api.documentation.annotations.RegistryDescription;
+import com.cleanroommc.groovyscript.helper.Alias;
+import com.cleanroommc.groovyscript.helper.SimpleObjectStream;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import net.minecraft.item.ItemStack;
+import essentialcraft.api.DemonTrade;
+import net.minecraftforge.fml.common.registry.EntityEntry;
+
+@RegistryDescription(category = RegistryDescription.Category.ENTRIES)

```

Comment:
yeah, i think you should add an admonition here - i think a `DANGER`-type would be good

Reply:
it's notable that the mod's former maintainer fixed this a few days ago, but will still add it